### ISSUE TITLE
Beta create entries page: Enable all panels by default

### ISSUE THREAD

**Issue by nfagerlund** [dreamwidth/dw-free#2652]
The hideable panel design is cool and good, but it's also minimally discoverable. 

The default set of panels for users who've never configured their preferences before excludes some things, including the entry display date. 

In order to keep people from wondering where their features went, we should default to showing everything and let people opt out of what they _don't_ want.

**Comment by nfagerlund** [comment]
Additional idea: we should add a do-nothing panel that just says "Hey, you can use the "settings" link up top to customize the layout of this page and hide panels you don't use!"

Alternately, if we want to continue hiding some panels by default, the instructions panel can call those out.

**Comment by rahaeli** [comment]
Huh. I like the idea of a dummy panel a lot, especially if it disappears after it's been dismissed or hidden! ...or maybe even put in a "first time visit" brief tour, or extend the messages thing that Momiji made to put up a banner up top that people could dismiss whenever they wanted to get rid of it? In the meantime I could edit the beta banner that's there already to include a pointer to Settings.

The reasoning for hiding some panels by default is those are the lesser-used settings on an entry -- it was part of trying to decomplicate the posting process both visually and workflow-wise. I don't remember the exact threshhold we used, but it was something like "check the $largenum most recently posted entries and hide the panel for any option that was used in fewer than 10% (or whatever) of them". I would like to keep hiding the lesser-used ones by default, because learning the site is still hella complex for new people and the less we present them with at once, the better, but you're right that people's eyes just gloss over the Settings link, and it's not super obvious that there are more options or that you can drag and rearrange.

**Comment by nfagerlund** [comment]
Yeah, once the dummy panel occurred to me, I got more OK with leaving things hidden by default. 

It looks like if JS is enabled, the dummy panel could dismiss itself by calling the same RPC endpoint the settings panel uses? Without JS, it'd have to just say "go to the settings page to remove this."

**Comment by rahaeli** [comment]
I will leave the actual technical solution to them what know more about frontend than I do, but let me know when you need it and I can write some copy.
